cylc/flow/scripts/show.py

```diff
@@ -258,12 +276,38 @@ def main(_, options, suite, *task_args):
                         ' (<red>- => not completed</red>):')
                     if not t_proxy['outputs']:
                         print('  (None)')
-                    for key, val in t_proxy['outputs'].items():
-                        print_msg_state(f'{task_id} {key}', val)
-                    if t_proxy['extras']:
-                        print('\nother:')
-                        for key, value in t_proxy['extras'].items():
-                            print('  o  %s ... %s' % (key, value))
+                    for output in t_proxy['outputs']:
+                        info = f'{task_id} {output["label"]}'
+                        print_msg_state(info, output['satisfied'])
+                    if (
+                            t_proxy['clockTrigger']['timeString']
+                            or t_proxy['externalTriggers']
+                            or t_proxy['xtriggers']
+                    ):
+                        if t_proxy['clockTrigger']['timeString']:
+                            state = t_proxy['clockTrigger']['satisfied']
+                            print(
+                                f'  o  Clock trigger time reached ... {state}')
```

Comment:
There is a `print_msg_state` which standardises message format as well as putting them in colour.

```suggestion
                            print_msg_state(f'  o  Clock trigger time reached', state)
```